scripts/provision-kind-cluster.sh

```diff
@@ -66,7 +74,7 @@ debug_msg "kind: using Kubernetes $K8_VERSION"
 echo -n "creating kind cluster $cluster_name ... "
 for i in $(seq 0 5); do
   if [[ -z $(kind get clusters 2>/dev/null | grep $cluster_name) ]]; then
-      kind create cluster -q --name "$cluster_name" --image $K8_VERSION --config "$KIND_CONFIG_FILE" --kubeconfig $TMP_DIR/kubeconfig 1>&2 || :
```

Comment:
@RedbackThomson was the above change on purpose or did you just forget to re-enable quiet-mode after debugging?

Reply:
I was initially going to put it back to quiet, but I have had lots of silent errors with it that I feel for the sake of debugging it would be better to leave it verbose. For instance, if it fails (silently) then "loading images into cluster" exits with code 2. That red herring led me on a trail of Google-fu until I finally checked that the cluster was actually standing.